### DISCUSSION TITLE
How do people get the "indented" current line number on the number line?

### DISCUSSION THREAD

**Discussion by fromrealstoreals** [LazyVim/LazyVim#4988] [Q&A]
Hi, I've scoured through the LazyVim docs and have not been able to find an answer. Currently, my line numbers look this:
<img width="28" alt="image" src="https://github.com/user-attachments/assets/60ce374d-59bc-4086-9b46-c2e846a87316">
As you can see, the current number line is flush with the other relative numbers. What I am trying to achieve is for the current number line to be offset (not flush) with the rest of the relative numbers. 

I had this work in my LazyVim configuration before but after manually setting `number` and `relativenumber`, it went away. Even after removing these from my `options.lua` it did not restore this behaviour. 

The screenshot below is from Elijah Manor's video on LazyVim as posted on the LazyVim "Getting Started" page and is what I'm trying to achieve:
<img width="46" alt="image" src="https://github.com/user-attachments/assets/d942c2a0-8cb6-472e-8846-36bf4db16034">
Does anyone know how to set this?

**Comment by folke** [comment]
snacks.statuscolumn
